vpr/src/place/initial_placement.cpp

```diff
@@ -445,8 +445,9 @@ void initial_placement(enum e_pad_loc_type pad_loc_type,
         place_ctx.block_locs[blk_id].loc = t_pl_loc();
     }
 
-    if (pad_loc_type == USER) {
-        read_user_pad_loc(pad_loc_file);
+    /*If the user specified block locations using a constraints file, read those locations in here*/
+    if (block_loc_type == LOCKED) {
+        read_user_block_loc(constraints_file);
```

Comment:
Change to just call read_user_block_loc (can have it return if constraints_file is NULL or an empty string).  Or could check before calling it with this if ().